data/tlg0007/tlg091/tlg0007.tlg091.perseus-eng3.xml

```diff
@@ -96,7 +96,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
+        	<p rend="indent"><said who="#Basilocles"><label>BASILOCLES.</label> You people have kept it up till well into the evening, Philinus, escorting the foreign visitor around among the statues and votive offerings. For my part, I had almost given up waiting for you.</said> </p> <p rend="indent"> <said who="#Philinus"><label>PHILINUS.</label> The fact is, Basilocles, that wre went slowly, sowing words, and reaping them straightway with strife, like the men sprung from the Dragon’s teeth, words with meanings behind them of the contentious sort, which sprang up and flourished along our way.</said> </p> <p rend="indent"> <said who="#Basilocles"><label>BASILOCLES.</label> Will it be necessary to call in someone else of those who were with you; or are you willing, as a favour, to relate in full what your conversation was and who took part in it?</said> </p> <p rend="indent"> <said who="#Philinus"><label>PHILINUS.</label> It looks, Basilocles, as if I shall have that to do. In fact, it would not be easy for you to find anyone of the others in the town, for I saw most of them once more on their way up to the Cory ei an cave and Lycoreia<note resp="editor" place="unspecified" anchored="true">Pausanias, x. 6. 2-3.</note> with the foreign visitor.</said> <pb xml:id="v.5.p.261"/> </p> <p rend="indent"> <said who="#Basilocles"><label>BASILOCLES.</label> Our visitor is certainly eager to see the sights, and an unusually eager listener.</said> </p> <p rend="indent"> <said who="#Philinus"><label>PHILINUS.</label> But even more is he a scholar and a student. However, it is not this that most deserves our admiration, but a winning gentleness, and his willingness to argue and to raise questions, which comes from his intelligence, and shows no dissatisfaction nor contrariety with the answers. So, after being with him but a short time, one would say, <q>O child of a goodly father!</q> <note resp="editor" place="unspecified" anchored="true"> <foreign xml:lang="lat">Cf.</foreign> Plato, <title rend="italic">Republic</title>, 368 a.</note> You surely know Diogenianus, one of the best of men.</said> </p> <p rend="indent"> <said who="#Basilocles"><label>BASILOCLES.</label> I never saw him myself, Philinus, but I have met many persons who expressed a strong approval of the man’s words and character, and who had other compliments of the same nature to say of the young man. But, my friend, what was the beginning and occasion of your conversation?</said> </p> </div> 
```

Comment:
I am having trouble finding any difference here?

Reply:
Friendly tip: using `git diff --word-diff-regex=.` makes these kind of changes much easier to see. However, the change in the above line is this:
```diff
- <label>BASILOCLES</label>.
+ <label>BASILOCLES.</label>
```